benchmark/statsaggregator.py

```diff
@@ -93,7 +98,14 @@ def aggregate_request(self, stats: RequestStats):
             if stats.response_status_code == 429:
                self.throttled_count += 1
          else:
-            self.request_latency._append(stats.request_start_time, stats.response_end_time - stats.request_start_time)
+            request_latency = stats.response_end_time - stats.request_start_time
+            self.request_latency._append(stats.request_start_time, request_latency)
+            if request_latency > self.window_duration:
+               logging.warning((
+                     f"request completed in {math.ceil(request_latency)} seconds, while aggregation-window is {int(self.window_duration)} "
+                     "seconds. Consider increasing aggregation-window to at least 2x your typical request latency."
```

Comment:
Since we started the log with small caps:
```suggestion
                     "seconds, consider increasing aggregation-window to at least 2x your typical request latency."
```